treadmill/infra/setup/master.py

```diff
@@ -0,0 +1,106 @@
+from treadmill.infra.vpc import VPC
```

Comment:
we do not import classes.

from treadmill.infra import vpc
...

x = vpc.VPC()